[IMP] MRP: updated Kits doc rst content

weka-odoo opened the pull request:
Went through the existing kits documentation and changed a lot of the content, as much of it was outdated. Also replaced the images with new ones.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/2720).

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,65 @@
-.. image:: kit_shipping/kit-bom.png
-    :align: center
+Using kits to manage complex BoMs 
+=================================
```
Make sure the `=` matches the last letter of the section title after you change `Using` to `Use`

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,65 @@
 
-Manage Stock of Kit Product and Component Products
-==================================================
+Kits are also used to manage *multilevel BoMs*. These are products that contain other products as
```
Replace `*multilevel BoMs*` with `` *multilevel* :abbr:`BoMs (Bills of Materials)` ``

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,65 @@
-Manage Stock of Kit Product and Component Products
-==================================================
+Kits are also used to manage *multilevel BoMs*. These are products that contain other products as
+components and therefore require nested BoMs. Incorporating pre-configured kits into multilevel BoMs
```
- Replace `BoMs` with `` :abbr:`BoMs (Bills of Materials)` ``
- Replace `multilevel BoMs` with `` multilevel :abbr:`BoMs (Bills of Materials)` `

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,65 @@
-==================================================
+Kits are also used to manage *multilevel BoMs*. These are products that contain other products as
+components and therefore require nested BoMs. Incorporating pre-configured kits into multilevel BoMs
+allows for cleaner organization. List a kit as a component in a higher-level product’s BoM to
```
- Add `` Under the :guilabel:`Components`, `` before `List a kit...`
- Uncapitalize the first letter of `List`
- Replace `product's BoM` with `` product's :abbr:`BoM (Bill of Materials)` ``

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,65 @@
+Kits are also used to manage *multilevel BoMs*. These are products that contain other products as
+components and therefore require nested BoMs. Incorporating pre-configured kits into multilevel BoMs
+allows for cleaner organization. List a kit as a component in a higher-level product’s BoM to
+eliminate the need to add the kit’s parts individually, as illustrated below by the Custom Computer
+Kit in the Custom Server BoM.
```
Delete `by the Custom Computer Kit in the Custom Server BoM`

mivu-odoo replied:
Does it matter what the "BoM Type" is for multilevel BoMs?

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,65 @@
 
-In that case the product type will be storable, and the BOM type either 'Manufacture this product' 
-or 'Subcontracting' as shown below.
+To access a comprehensive overview of the multilevel BoM's components, click on the
```
Replace `BoM's` with `` :abbr:`BoM's (Bill of Material's)` ``

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,65 @@
-In that case the product type will be storable, and the BOM type either 'Manufacture this product' 
-or 'Subcontracting' as shown below.
+To access a comprehensive overview of the multilevel BoM's components, click on the
+:guilabel:`Structure & Cost` smart button. Sublevel BoMs can be expanded and viewed from this
```
Replace `BoMs` with `` :abbr:`BoMs (Bills of Materials)` ``

mivu-odoo reviewed (changes requested):
Hi @weka-odoo!

Thank you for your patience on this!

I had some suggestions and questions for parts that needed more clarity. Let me know if you need me to explain any of my comments!

When you're ready, please tag me again for another look. Thank you :smile_cat:

weka-odoo commented:
@mivu-odoo Here are the incorporated edits. A few things:
* I deleted part of a sentence referring to the image because Zac mentioned in a previous PR that the images should be standalone. 
* I added a note at the bottom of the doc with a link to the semi-finished products user doc (I'm currently editing this one), since similar (but distinct!) material is discussed in both docs. 

Please re-review the changes whenever you are able to. Thank you!

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,76 @@
+
+To use a kit as a sellable product or simply as a component organization tool, the kit should first
+be created as a product. To create a kit product, go to :menuselection:`Manufacturing or Inventory
+--> Products --> Products`, then click :guilabel:`Create`.
```
Add `and` before `then click`

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,76 @@
+
+After fully configuring the kit product and its components, create a :abbr:`BoM (Bill of
+Materials)` for the kit product. Go to :menuselection:`Manufacturing --> Products --> Bills of
+Materials`, then click :guilabel:`Create`. Next, set the :guilabel:`Product` field to the
```
Add `and` before `then click`

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,76 @@
+   :align: center
+   :alt: Kit as a component in a multilevel bill of materials.
+
+To access a comprehensive overview of the multilevel :abbr:`BoM's (Bills of Material's)`
```
Remove the `s` from `Bills`

mivu-odoo commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,76 @@
+
+.. note::
+   Kits are primarily used to bundle components together for organization or sale. To manage
+   multilevel products that require manufactured subcomponents, refer to :doc:`this documentation
```
Add `on sub-assemblies` after `this documentation`

mivu-odoo reviewed (approved):
Hi @weka-odoo!

Thank you for your hard work on this!

I found a few small edits. Once you integrate those changes, feel free to go straight to ZST for final technical review. No need to tag me again :smile_cat:

weka-odoo commented:
@StraubCreative This PR is ready for you to review whenever you get the chance :)

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,76 @@
-in to the work order list of the main component. 
\ No newline at end of file
+========
+Use Kits
```
Sentence case.
```suggestion
Use kits
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,76 @@
+
+Next, set the :guilabel:`Product Type` to :guilabel:`Storable Product`: this allows for management
+of kit inventory. The :guilabel:`Route` designation under the :guilabel:`Inventory` tab does not
+matter, since Odoo uses the routes of the kit’s components for replenishment purposes. All other
```
replace serif apostrophe (which was likely copied over from a gdoc) with sans-serif version so it's more RST-friendly.
```suggestion
matter, since Odoo uses the routes of the kit's components for replenishment purposes. All other
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,76 @@
+that contain other products as components and therefore require nested :abbr:`BoMs (Bills of
+Materials)`. Incorporating pre-configured kits into multilevel :abbr:`BoMs (Bills of Materials)`
+allows for cleaner organization of bundled products. Under :guilabel:`Components`, list a kit as a
+component in a higher-level product’s :abbr:`BoM (Bills of Material)` to eliminate the need to add
+the kit’s parts individually. Any :guilabel:`BoM Type` can be used for the higher-level product's
```
apostrophes
```suggestion
component in a higher-level product's :abbr:`BoM (Bills of Material)` to eliminate the need to add
the kit's parts individually. Any :guilabel:`BoM Type` can be used for the higher-level product's
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:
```diff
@@ -1,87 +1,76 @@
+
+When creating a manufacturing order for a product with a multilevel :abbr:`BoM (Bill of
+Materials)`, the kit product automatically expands to show all components. Any operations in the
+kit’s :abbr:`BoM (Bill of Materials)` are also added to the list of work orders on the
```
apostrophe
```suggestion
kit's :abbr:`BoM (Bill of Materials)` are also added to the list of work orders on the
```

StraubCreative reviewed (approved):
This is a great PR, nice job @weka-odoo 👍 

Two small things to keep an eye on for next time:
- Sentence case for h1 tag
- serif vs. sans-serif apostrophe (referenced [here](https://github.com/odoo/documentation/pull/2925#pullrequestreview-1185456084) on another PR)

StraubCreative commented:
Hi @odoo/doc-review this doc's ready for your review :)

AntoineVDV reviewed (approved):
@robodoo r+

weka-odoo commented:
@fw-bot r+
